test/test_parmed_structure_slicing.py

```diff
@@ -238,6 +239,46 @@ def test_chain_residue_atom_selection(self):
         self.assertRaises(ValueError, lambda:
                 pdb2[list(range(len(pdb2.atoms)+1))])
         self.assertRaises(ValueError, lambda: pdb2[[0,len(pdb2.atoms)]])
+        
+    def test_structure_box_and_space_group_and_symmetry(self):
+        def assert_correctly_copy(parm):
+            sliced_parm = parm['@1-3']
+            np.testing.assert_equal(parm.box, sliced_parm.box)
+            self.assertEqual(parm.space_group, sliced_parm.space_group)
+            if parm.symmetry is not None:
+                np.testing.assert_equal(parm.symmetry.data, sliced_parm.symmetry.data)
+            else:
+                self.assertTrue(sliced_parm.symmetry is None)
+        # pdb
+        parm = pmd.load_file(get_fn('4lzt.pdb'))
+        assert_correctly_copy(parm)
+        # pdb from rcsb
+        parm = pmd.download_PDB('2igd')
+        assert_correctly_copy(parm)
+        # cif
+        parm = pmd.load_file(get_fn('sample.cif'))
+        assert_correctly_copy(parm)
+        # LES parm7, no rst7
+        parm = pmd.load_file(get_fn('4lzt.les.parm7'))
+        assert_correctly_copy(parm)
+        # LES parm7, with rst7
+        parm = pmd.load_file(get_fn('4lzt.les.parm7'), xyz=get_fn('4lzt.les.rst7'))
+        assert_correctly_copy(parm)
+        # gro
+        parm = pmd.load_file(get_fn('1aki.charmm27.solv.gro'))
+        assert_correctly_copy(parm)
+        # amoeba: does slicing in original code works?
```

Comment:
Ah, then it doesn't work.  Not worth fixing since Amoeba topology files are rare and I think development on pmemd.amoeba is winding down and/or ending completely.